charts/generic-prometheus-alerts/templates/aws-elasticache-alerts.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if .Values.elastiCacheAlertsClusterIds -}}
+  {{- $targetNamespace := .Release.Namespace }}
+  {{- $targetApplication := required "A value for targetApplication must be set" .Values.targetApplication }}
+  {{- $elastiCacheAlertsEngineCPUThreshold := .Values.elastiCacheAlertsEngineCPUThreshold }}
+  {{- $elastiCacheAlertsEngineCPUThresholdMinutes := .Values.elastiCacheAlertsEngineCPUThresholdMinutes }}
+  {{- $elastiCacheAlertsCPUThreshold := .Values.elastiCacheAlertsCPUThreshold }}
+  {{- $elastiCacheAlertsCPUThresholdMinutes := .Values.elastiCacheAlertsCPUThresholdMinutes }}
+  {{- $elastiCacheAlertsFreeMemoryThreshold := .Values.elastiCacheAlertsFreeMemoryThreshold }}
+  {{- $elastiCacheAlertsFreeMemoryThresholdMinutes := .Values.elastiCacheAlertsFreeMemoryThresholdMinutes }}
+  {{- $targetApplicationBusinessHours := printf "and ON() %s:business_hours" .Values.targetApplication | replace "-" "_" }}
+  {{- $businessOrAllHoursExpression := ternary $targetApplicationBusinessHours "" .Values.businessHoursOnly}}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ $targetApplication }}-elasticache
+  labels:
+  {{- include "generic-prometheus-alerts.labels" . | nindent 4 }}
+spec:
+  groups:
+    - name: {{ $targetApplication }}-elasticache
+      rules:
+        {{- range $key, $val := $.Values.elastiCacheAlertsClusterIds }}
+        - alert: elasticache-enginecpu-utilisation
+          annotations:
+            message: elasticache cluster {{ $val }} in {{ $targetNamespace }} - CPU utilisation at {{`{{`}} printf "%.2f" $value {{`}}`}} which is over threshold of {{ $elastiCacheAlertsEngineCPUThreshold }} for last {{ $elastiCacheAlertsEngineCPUThresholdMinutes }} mins. This alert configured by app {{ $targetNamespace }}/{{ $targetApplication }}.
+            runbook_url: {{ $.Values.runbookUrl }}elasticache-enginecpu-utilisation
+            {{/* TODO*/}}
+            dashboard_url: {{ $.Values.grafanaUrl }}/d/VR46pmwWk/aws-elasticache?orgId=1&var-datasource=Cloudwatch&var-region=default&var-elasticacheclusteid={{ $key }}&from=now-6h&to=now
+            summary: elasticache cluster CPU usage too high
+          expr: |-
+            aws_elasticache_engine_cpuutilization_maximum{cache_cluste_id='{{ $key }}'} > {{ $elastiCacheAlertsEngineCPUThreshold }}
```

Comment:
`cache_cluste_id`  spelling error and see other occurances.

Reply:
updated the typo error